### ISSUE TITLE
Error with botocore == 1.34.47

### ISSUE THREAD

**Issue by termit-uanic** [binxio/aws-ssm-copy#35]
I'm sure this is not your problem, but I want to let you know that with `botocore==1.34.47` `aws-ssm-copy` stopped working. With `botocore==1.34.46` everything works.

Perhaps something can be done on your end.

The error itself looks like this:
```
# aws-ssm-copy -r --keep-going --target-path /fr-417-feature/v2-frontend /staging/v2-frontend
Traceback (most recent call last):
  File "/apps/.venv/bin/aws-ssm-copy", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/apps/.venv/lib/python3.11/site-packages/aws_ssm_copy/ssm_copy.py", line 336, in main
    cp.main()
  File "/apps/.venv/lib/python3.11/site-packages/aws_ssm_copy/ssm_copy.py", line 319, in main
    self.copy(
  File "/apps/.venv/lib/python3.11/site-packages/aws_ssm_copy/ssm_copy.py", line 196, in copy
    self.target_ssm.put_parameter(**parameter)
  File "/apps/.venv/lib/python3.11/site-packages/botocore/client.py", line 553, in _api_call
    return self._make_api_call(operation_name, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/apps/.venv/lib/python3.11/site-packages/botocore/client.py", line 962, in _make_api_call
    request_dict = self._convert_to_request_dict(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/apps/.venv/lib/python3.11/site-packages/botocore/client.py", line 1036, in _convert_to_request_dict
    request_dict = self._serializer.serialize_to_request(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/apps/.venv/lib/python3.11/site-packages/botocore/validate.py", line 381, in serialize_to_request
    raise ParamValidationError(report=report.generate_report())
botocore.exceptions.ParamValidationError: Parameter validation failed:
Unknown parameter in input: "ARN", must be one of: Name, Description, Value, Type, KeyId, Overwrite, AllowedPattern, Tags, Tier, Policies, DataType
```

Installed package versions:
```
# pip list
Package         Version
--------------- -------
aws-ssm-copy    0.5.2
boto3           1.34.47
botocore        1.34.47
jmespath        1.0.1
pip             23.2.1
python-dateutil 2.8.2
s3transfer      0.10.0
setuptools      65.5.0
six             1.16.0
urllib3         2.0.7
```

**Comment by mvanholsteijn** [comment]
I could reproduce this issue. Made a fix. Just waiting for the release pipeline to complete.

**Comment by mvanholsteijn** [comment]
Fixed in 0.5.3; Thank you for reporting.
